src/ExpenseTrackerGrupo2/Persistence/Database/Scripts/init.sql

```diff
@@ -0,0 +1,76 @@
+CREATE TABLE IF NOT EXISTS users (
+    user_id UUID PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL UNIQUE,
+    password_hash VARCHAR(100) NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS expenses (
+    expense_id UUID PRIMARY KEY,
+    user_id UUID NOT NULL,
+    amount DECIMAL(10, 2) NOT NULL,
+    description VARCHAR(255),
+    category VARCHAR(100),
+    expense_date DATE NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (user_id) REFERENCES users(user_id) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS incomes (
+    income_id UUID PRIMARY KEY,
+    user_id UUID NOT NULL,
+    amount DECIMAL(10, 2) NOT NULL,
+    source VARCHAR(100),
+    income_date DATE NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (user_id) REFERENCES users(user_id) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS budgets (
+    budget_id UUID PRIMARY KEY,
+    user_id UUID NOT NULL,
+    month INT NOT NULL,
```

Comment:
Are you sure about `INT` type in month column?

In requirements document says in this format should be saved. I think it would better `DATE`.
Request body: 
```
{ 
"budgetAmount": 1000.0, 
"month": "yyyy-mm" 
}
```

Reply:
It will be Date, but date its "yyyy-mm-dd" or varchar(7) "yyyy-mm" format

Reply:
Let's change it to varchar(7), okay? I think it could be a good option but in a future we can changed.

Reply:
Done